integreat_compass/cms/forms/registration_form.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from django import forms
+from django.contrib.auth.password_validation import validate_password
+from django.utils.translation import gettext as _
+
+from ..constants import group_names
+from ..models import User
+from .custom_model_form import CustomModelForm
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationForm(CustomModelForm):
```

Comment:
Ah ok, I see. I guess the `BaseUserCreationForm` could still be used since it does not handle the username (and we can still extend the email and display name validation), but ok we don't save that much code, just a few lines of password handling...